assets/scss/app.scss

```diff
@@ -77,4 +77,14 @@ table.tenant-details {
   th {
     font-weight: 600;
   }
-}
\ No newline at end of file
+}
+
+ul {
+  @extend .text;
+  @extend .list;
+  @extend .list-bullet;
+}
+
+.no-margin-bottom {
+  margin-bottom: 0;
+}
```

Comment:
this is needed because I can't have one of the bullets inside the markdown, see above

Reply:
ooooh, not terribly semantic

Reply:
fair point, what would you call it no-new-line?